pvlib/pvsystem.py

```diff
@@ -142,19 +142,18 @@ class PVSystem:
 
     module : string, optional
         The model name of the modules.
-        May be used to look up the module_parameters dictionary
-        via some other method.
 
-    module_type : string, default 'glass_polymer'
-         Describes the module's construction. Valid strings are 'glass_polymer'
-         and 'glass_glass'. Used for cell and module temperature calculations.
+    module_type : string, default `glass_polymer`
+         Describes the module's construction. Valid strings are `glass_polymer`
+         and `glass_glass`. Used for cell and module temperature calculations.
```

Comment:
If we want to use backticks for string literals in the parameter descriptions, I suggest we use double backticks with quote delimiters, like this: ``` ``'glass_glass'`` ```